motes.md

```diff
@@ -0,0 +1,42 @@
+* `BLS_WITHDRAWAL_PREFIX`
+  * Why int rather than bytes?
+* `MIN_SEED_LOOKAHEAD`
+  * Is this actually tunable?
+  * If so, what are the reprecussions?
+* `ACTIVATION_EXIT_DELAY`
+  * Reaquaint with purpose
+* AttesterSlashings
+  * `MAX_ATTESTER_SLASHINGS` is 1.
+  * Are there scenarios in which validators can create more effective slashable
+    messages than can be included on chain? For example, Validators split up to
+    create double attestations for checkpoints but different (junk) crosslink
+    data to prevent them from being aggregatable to the fullest
```

Comment:
I remember thinking about this. One way to create more effective slashable messages is as follows. Let's say you control `n` validators, and you want to make bad attestations `a_1`, ..., `a_n` for each of these validators. The attacker can broadcast aggregated attestations `(a_1, a_i)` for `i > 1` and never reveal the individual `a_i`. This is an argument for generalising the bitfield to a multibitfield.